presto-orc/src/main/java/com/facebook/presto/orc/Stripe.java

```diff
@@ -58,6 +61,11 @@ public InputStreamSources getDictionaryStreamSources()
         return dictionaryStreamSources;
     }
 
+    public LongDictionaryProvider getIntegerDictionaryProvider()
```

Comment:
method name should getLongDictionaryProvider()

Reply:
Surprised that the built-in refactor would miss that. 😂